### PR TITLE
Update intro-to-data-structures.md

### PR DESCRIPTION
**Pull request by vaishali614**
Addresses #570

### CODE REVIEW

**Comment by vaishali614** [comment]
I've made changes to the slide: https://github.com/Techtonica/curriculum/blob/master/data-structures/intro-to-data-structures.md from issue #570

File: data-structures/intro-to-data-structures.md [hunk 1]
```diff
@@ -70,3 +74,4 @@ Using a timer, pair up for 5 minutes and quiz each other while reviewing the dif
 - [Watch Data Structures and Algorithms in action at visualgo.net](https://visualgo.net/en)
 - [Article w/Videos:  Data Structures and Algorithms in JavaScript by Beau Carnes](https://medium.freecodecamp.org/10-common-data-structures-explained-with-videos-exercises-aaff6c06fb2b)
 - [Article: https://www.tutorialspoint.com/data_structures_algorithms/data_structures_basics.htm](https://www.tutorialspoint.com/data_structures_algorithms/data_structures_basics.htm)
```
Comment (alodahl) [inline data-structures/intro-to-data-structures.md]: Since tutorials point is in the lesson, please move the link from supplemental to “materials”.

Comment (vaishali614) [reply data-structures/intro-to-data-structures.md]: okayy! i've made the required changes. please check.

File: data-structures/intro-to-data-structures.md [hunk 2]
```diff
@@ -70,3 +74,4 @@ Using a timer, pair up for 5 minutes and quiz each other while reviewing the dif
...
+- [Practice Data Structures](https://www.hackerearth.com/practice/data-structures/arrays/1-d/tutorial/)
```
[the remark this answers is not in the document]
Comment (vaishali614) [reply data-structures/intro-to-data-structures.md]: I've removed it and put it in another section. Please check.

**Pull request merged**